src/v2/Apps/Conversation/Components/__tests__/Conversation.jest.tsx

```diff
@@ -1,89 +1,134 @@
-import { mount } from "enzyme"
 import React from "react"
-import { Item } from "../Item"
-import { Conversation_conversation } from "v2/__generated__/Conversation_conversation.graphql"
+import { ConversationPaginationContainer } from "../Conversation"
+import {
+  InquiryImage,
+  MockedConversation,
+  MockedInquiryConversation,
+} from "v2/Apps/__tests__/Fixtures/Conversation"
+import { MockBoot, renderRelayTree } from "v2/DevTools"
+import { graphql } from "react-relay"
+import {
+  ConversationPaginationTestQueryRawResponse,
+  ConversationPaginationTestQueryResponse,
+} from "v2/__generated__/ConversationPaginationTestQuery.graphql"
+import { useTracking } from "react-tracking"
+import { SystemContextProvider } from "v2/System"
+import { HeadProvider } from "react-head"
 
-describe("Conversation", () => {
-  describe("Item", () => {
-    describe("when inquiry item is an artwork", () => {
-      // @ts-expect-error STRICT_NULL_CHECK
-      const artworkItemProps: Conversation_conversation["items"][0]["item"] = {
-        __typename: "Artwork",
-        id: "12345",
-        date: "June 22, 2020",
-        title: "Untitled",
-        artistNames: "Banksy",
-        href: "site.com/banksy",
-        image: {
-          url: "image.com/banksy-image",
-        },
-        listPrice: {
-          __typename: "Money",
-          display: "$2000",
-        },
+jest.unmock("react-relay")
+
+const render = (
+  node: ConversationPaginationTestQueryRawResponse["node"],
+  user: User
+) =>
+  renderRelayTree({
```

Comment:
Is it possible to use our modern relay test tools and patterns? See the [Best Practices](https://github.com/artsy/force/blob/master/docs/best_practices.md#write-unit-tests-for-new-components) doc for more info.

Reply:
@damassi yeah we chatted about that. I think we need to refactor this component in order to do so. @copperkraft has a work in progress PR for [this](https://github.com/artsy/force/pull/8122). I'm creating a ticket for us to do that this week. I felt like the outdated test pattern was better than no coverage at all.

Reply:
Cool, that's what I figured 👍 np!